Documentation/Usage.md

```diff
@@ -115,204 +115,257 @@ url.
 Example of a published package:
 https://github.com/apple/example-package-fisheryates
 
-## Require System Libraries
+## Import system libraries
 
-You can link against system libraries using the package manager. To do so, there
-needs to be a special package for each system library that contains a modulemap
-for that library. Such a wrapper package does not contain any code of its own.
+As of Swift 4.2, the package manager is capable of importing C libraries from the system without requiring a special wrapper package. We’ll import [Cairo](cairographics.org), a popular 2D vector graphics library, as an example. This guide assumes you already know how to include and link a system library in a C or C++ project.
 
-Let's see an example of using [libgit2](https://libgit2.github.com) from an
-executable.
+For our example, we’ll create a new package called `example`:
 
-First, create a directory called `example`, and initialize it as a package that
-builds an executable:
+```bash 
+$ mkdir example 
+$ cd example 
+$ swift package init --type executable
+```
 
-    $ mkdir example
-    $ cd example
-    example$ swift package init --type executable
+The `example` folder should look something like this:
 
-Edit the `Sources/main.swift` so it consists of this code:
+```
+.
+├── Package.swift
+├── README.md
+└── Sources
+    └── example
+        └── main.swift
```

Comment:
The latest SwiftPM also generates unit tests for executable targets.

Reply:
i deleted the `Tests/` directory because it wasn’t relevant to the guide and it was cluttering up the directory tree diagrams